src/phoneField/index.tsx

```diff
@@ -120,16 +122,19 @@ export default class PhoneField extends PureComponent<PhoneFieldProps, PhoneFiel
     countriesWhiteList: this.props.countriesWhiteList,
     [FIELDS.PHONENUMBER]: this.props.defaultValue,
     [FIELDS.PHONEPREFIX]: iso2toDialCode(this.props.defaultPrefix),
+    phoneRegion: [this.props.defaultPrefix],
```

Comment:
if you don't find the phoneRegion, maybe you should add a default value ([])

Reply:
It's already the case on defaultProps >> `defaultPrefix: '',`
And into the onChange function >> `phoneRegion: !isEmpty(iso2Codes) ? iso2Codes : [''],`

Reply:
Done